tests/nlu/base/test_evaluation.py

```diff
@@ -270,7 +270,9 @@ def test_run_cv_evaluation():
     nlu_config = config.load("sample_configs/config_pretrained_embeddings_spacy.yml")
 
     n_folds = 2
-    intent_results, entity_results = cross_validate(td, n_folds, nlu_config)
+    intent_results, entity_results, response_selection_results = cross_validate(
```

Comment:
@wochinge What would be the best file to add the test for `is_response_selector_present` function?
As I understand from the code, `Interpreter` is returned by `Trainer.train` which means in the test we need to load a sample config, data and pass them to `Trainer.train` and collect the interpreter which gets tested inside `is_response_selector_present`. If it is implemented this way, can the test go in `tests/nlu/base/test_interpreter.py`?

Reply:
Or is there a way, by which I can create an instance of `Interpreter` by just passing a config and not calling `Trainer.train()`. I guess that isn't possible because by design `Interpreter` is returned by `Trainer`

Reply:
Since `is_response_selector` present is part of the `rasa.nlu.test` module, I think it should go to `tests.nlu.base.test_evaluation`

Can't you just call `Interpreter(...)` with a list of components? So without training anything?

Reply:
Yes, that works. Thanks!